libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat

```diff
@@ -61,8 +62,8 @@ PB8 CAN1_RX CAN1
 PB9 CAN1_TX CAN1
 
 # debugger support, disabled as PA13 used for LED
```

Comment:
You should also remove this line

Reply:
Thanks to that I will make the changes.
Should I make the changes and create a new pull request (to keep fewer commits)?
or should I add the changes with new commits here itself?
What is preferred?

Reply:
@Lokesh-Carbonix  Makes the changes as an additional commit and then squash the commits to one commit.

Just a note. We consider a manufacturers HWDef to be theirs. AKA we just trust you to get it correct. Especially for boards in progress like I think this one is?

Changes to boards out in general use get more scrutiny.

Reply:
Thanks, @hendjoshsr71 I will squash it. Yes, I understand on HWdef is the manufacturer's responsibility to test and this board is at an almost stable state.